From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 21 Jun 2018 23:45:23 -0400
Subject: look for sort instead of zip during unit tests

On debian, sort is part of coreutils.  I think it ships by default on
any modern MacOS system.  And SORT.EXE is available on DOS and Windows
too (https://www.computerhope.com/sorthlp.htm).  zip, on the other
hand, isn't installed by default on debian, and i don't know that it's
available on Windows.

Making this change should still work to exercise the test, but without
making zip a dependency for the test suite itself.
---
 package/tests/gpgAgent-test.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/tests/gpgAgent-test.js b/package/tests/gpgAgent-test.js
index 902bd31..daefe8b 100644
--- a/package/tests/gpgAgent-test.js
+++ b/package/tests/gpgAgent-test.js
@@ -163,8 +163,8 @@ test(withEnigmail(function resolveToolPathDefaultValues(enigmail) {
   withEnvironment({}, function(e) {
     resetting(EnigmailGpgAgent, 'agentPath', "/usr/bin/gpg-agent", function() {
       enigmail.environment = e;
-      var result = EnigmailGpgAgent.resolveToolPath("zip");
-      Assert.equal("zip", result.leafName.substr(0, 3));
+      var result = EnigmailGpgAgent.resolveToolPath("sort");
+      Assert.equal("sort", result.leafName.substr(0, 4));
     });
   });
 }));
